build-packages.cmd

```diff
@@ -60,6 +60,15 @@ if NOT [!ERRORLEVEL!]==[0] (
   exit /b 1
 )
 
+rem Build the TestHost package
+set __msbuildArgs="%__ProjectDir%\src\.nuget\Microsoft.NETCore.TestHost\Microsoft.NETCore.TestHost.builds" !allargs!
```

Comment:
Is the idea of this line to enforce building that package? packages.builds should be in charge of building all the packages.
As @wes points out, the script should only call one project and should avoid to have logic on it. With the changes Priya is doing we are going to be more close to that stage.
Independently of PR #6400, this change in the scripts should be avoided.